### ISSUE TITLE
Can't generate schema from openApi spec with "oneOf"

### ISSUE THREAD

**Issue by kargalskaya** [IBM/openapi-to-graphql#289]
Hi!
Trying to generate schemas from openapi and get error
**OpenAPI-to-GraphQL creation event error:  Cannot process schema '{"oneOf":[{"$ref":"#/components/schemas/Male"},{"$ref":"#/components/schemas/Female"}]}'. Cannot identify type of schema.**
```json
{
  "openapi" : "3.0.1",
  "info" : {
    "title" : "rbp-employee-service",
    "version" : "0.1"
  },
  "paths" : {
    "/employees" : {
      "get" : {
        "operationId" : "getEmployees",
        "parameters" : [
          {
            "name" : "userId",
            "in" : "query",
            "required" : true,
            "schema" : {
              "type" : "string"
            }
          }
        ],
        "responses" : {
          "200" : {
            "description" : "",
            "content" : {
              "application/json" : {
                "schema" : {
                  "type" : "array",
                  "items" : {
                    "$ref" : "#/components/schemas/Employee"
                  }
                }
              }
            }
          }
        }
      }
    }
  },
  "components" : {
    "schemas" : {
      "Employee" : {
        "type" : "object",
        "properties" : {
          "id" : {
            "type" : "string"
          },
          "details" : {
            "$ref" : "#/components/schemas/Details"
          }
        }
      },
      "Female" : {
        "type" : "object"
      },
      "Details" : {
        "type" : "object",
        "properties" : {
          "gender" : {
            "$ref" : "#/components/schemas/Gender"
          }
        }
      },
      "Gender" : {
        "oneOf" : [
          {
            "$ref" : "#/components/schemas/Male"
          },
          {
            "$ref" : "#/components/schemas/Female"
          }
        ]
      },
      "Male" : {
        "type" : "object"
      }
    }
  }
}
```

**Comment by Alan-Cha** [comment]
@kargalskaya Hi! We have a big update hopefully coming out early this week that will address these problems and bring a long a number of improvements.

**Comment by Alan-Cha** [comment]
@kargalskaya Just released 2.0.0 and it handles this case! 

Just for reference, the `gender` field resolves into the [arbitrary JSON type](https://github.com/taion/graphql-type-json). This is because the `Male` and `Female` schemas do not contain any properties. With v2.0.0, `oneOf` should resolve to a union type but in this case, it has to fall back onto the JSON type.
